scripts/prettier/index.js

```diff
@@ -28,35 +29,26 @@ const defaultOptions = {
 };
 const config = {
   default: {
-    patterns: [
-      // Internal forwarding modules
-      'packages/*/*.js',
-      // Source files
-      'packages/*/src/**/*.js',
-      'packages/shared/**/*.js',
-    ],
+    patterns: sourcePath,
     ignore: ['**/node_modules/**'],
   },
   scripts: {
-    patterns: [
-      // Forwarding modules that get published to npm (must be ES5)
-      'packages/*/npm/**/*.js',
-      // Need to work on Node
-      'scripts/**/*.js',
-      'fixtures/**/*.js',
-    ],
+    patterns: [...npmPath, ...nodePath],
     ignore: [
       '**/node_modules/**',
       // Built files and React repo clone
       'scripts/bench/benchmarks/**',
+      // shims & flow treat as ESNext
+      'scripts/flow/*.js',
+      'scripts/rollup/shims/**/*.js',
```

Comment:
It feels confusing that we still have to explicitly ignore some paths here but ESLint is fully covered by just “patterns” config.